docs/usage.rst

```diff
@@ -242,6 +242,29 @@ a version:
     >>> semver.bump_build("3.4.5-pre.2+build.4")
     '3.4.5-pre.2+build.5'
 
+    
+Increasing Parts of a Version Taking into Account Prereleases
+-------------------------------------------------------------
+
+.. versionadded:: 2.9.2
+   Added :func:`semver.next_version`.
+
+If you want to raise your version and take prereleases into account,
+the :func:`semver.next_version` or :func:`semver.VersionInfo.next_version`
+would perhaps a better fit.
+
+
+.. code-block:: python
+
+    >>> semver.next_version("3.4.5-pre.2+build.4", part="prerelease"))
+    '3.4.5-pre.3'
+    >>> semver.next_version("3.4.5-pre.2+build.4", part="patch")
+    '3.5.0'
```

Comment:
Is it correct value? If I understand problem correctly, it should be
```suggestion
    '3.4.5'
```
or `part="minor"`

Reply:
@ppkt Right Karol! Thanks for the tip! :+1: